Packs/EmailCommunication/ReleaseNotes/1_3_2.md

```diff
@@ -1,11 +1,12 @@
 
 #### Incident Fields
-- **Email Code Generator**
-- **incidentfield-emailcodegenerator_CHANGELOG.md**
+- **Email Generated Code**
+- **incidentfield-emailgeneratedcode_CHANGELOG.md**
 
 #### Scripts
 ##### SendEmailReply
 - Updated the script to use a UUID for every new incident that is created.
 ##### PreprocessEmail
 - Updated the script to use a UUID for every new incident that is created.
+- To avoid performance issues, limited the time to look for related incidents to 2 months.
```

Comment:
- `**Breaking Change** To avoid performance issues, limited the time to look for related incidents to 2 months.`